warehouse/query-core/src/main/java/datawave/query/language/functions/jexl/UniqueByTenth.java

```diff
@@ -0,0 +1,22 @@
+package datawave.query.language.functions.jexl;
+
+import datawave.query.jexl.functions.QueryFunctions;
+import datawave.query.jexl.visitors.QueryOptionsFromQueryVisitor;
+import datawave.query.language.functions.QueryFunction;
+
+import java.util.ArrayList;
+
+/**
+ * Function to return a unique result for every hour of the day for a given list of fields. This function is equivalent to {@code #unique(field[DAY])}.
+ */
+public class UniqueByTenth extends UniqueByFunction {
```

Comment:
Might be worth also renaming this to `UniqueByTenthOfHour`

Reply:
Ok, done